src/commands/deploy/utils.ts

```diff
@@ -742,7 +742,7 @@ export async function uploadEnvVarsFromFile(
                 .then(async () => {
                     const envVarKeys = envVars.map((envVar) => envVar.name);
                     log.info(
-                        `The following environment variables ${envVarKeys.join(", ")} were uploaded to the project successfully.`,
+                        `The following environment variables - ${envVarKeys.join(", ")} - were uploaded to the project successfully.`,
```

Comment:
I think it would be easier to read if the phrase wouldn't be splitted in half.

```
The following environment variables were uploaded to the project successfully:

${envVarKeys.join(", ")}
```